inspirehep/utils/record.py

```diff
@@ -29,36 +29,18 @@
 
 def get_title(record):
     """Get preferred title from record."""
-    chosen_title = ""
-
-    for title in record.get('titles', []):
-        if 'source' in title and title.get('source') != 'arXiv':
-            return title.get('title')
-        elif 'source' in title and title.get('source') == 'arXiv':
-            pass
-        else:
-            chosen_title = title.get('title')
-
-    if not chosen_title and len(record.get('titles', [])) > 0:
-        chosen_title = record['titles'][0]['title']
-
-    return chosen_title
+    if record.get('titles', []):
+        return record['titles'][0].get('title', '')
+    else:
+        return ""
 
 
 def get_subtitle(record):
     """Get preferred subtitle from record."""
-    chosen_subtitle = ""
-
-    for title in record.get('titles', []):
-        if 'source' in title and 'subtitle' in title and title.get('source') != 'arXiv':
-            return title.get('subtitle')
-        elif 'source' in title and 'subtitle' in title and title.get('source') == 'arXiv':
-            if not chosen_subtitle:
-                chosen_subtitle = title.get('subtitle')
-        else:
-            if 'subtitle' in title:
-                chosen_subtitle = title.get('subtitle')
-    return chosen_subtitle
```

Comment:
What do you mean existing records? What this PR does is to put the 245 (i.e. the main title) at the beginning of the `titles` field, and I have consequently amended the `get_(sub)title` functions to reflect this. So everything should be correct. Or I am not understanding what you mean :-)